### PR TITLE
Exception Logging for Covid19 Sample Importer

### PR DESCRIPTION
**Pull request by DavidStreid**
* Separating out Exceptions w/ different error messages
* Adding Stack Trace

### CODE REVIEW

File: src/main/java/com/velox/sloan/cmo/workflows/covid19/Covid19SampleImporter.java [hunk 1]
```diff
@@ -78,9 +79,17 @@ public PluginResult run() throws ServerException {
                 return new PluginResult(false);
             }
 
-        } catch (Exception e){
-            String message = String.format("Error while importing new COVID-19 Samples into LIMS\n%s", e.getMessage());
-            logError(message);
+        } catch (ServerException e){
+            String message = String.format("Server Exeception while importing new COVID-19 Samples into LIMS\n%s", e.getMessage());
```
Comment (dmcmanam) [inline src/main/java/com/velox/sloan/cmo/workflows/covid19/Covid19SampleImporter.java:83]: typo

File: src/main/java/com/velox/sloan/cmo/workflows/covid19/Covid19SampleImporter.java [hunk 2]
```diff
@@ -78,9 +79,17 @@ public PluginResult run() throws ServerException {
...
+            logError(message, e);
+            return new PluginResult(false);
+        } catch (IOException e) {
+            String message = String.format("IOException while importing new COVID-19 Samples into LIMS\n%s", e.getMessage());
+            logError(message, e);
```
Comment (dmcmanam) [inline src/main/java/com/velox/sloan/cmo/workflows/covid19/Covid19SampleImporter.java:88]: or similarly - logError(String.format("IOException while importing new COVID-19 Samples into LIMS with message:", e.getMessage(), e);

Comment (DavidStreid) [reply src/main/java/com/velox/sloan/cmo/workflows/covid19/Covid19SampleImporter.java:88]: will do

File: src/main/java/com/velox/sloan/cmo/workflows/covid19/Covid19SampleImporter.java (hunk 1 above)
Comment (DavidStreid) [reply src/main/java/com/velox/sloan/cmo/workflows/covid19/Covid19SampleImporter.java:83]: ahh fixed

**Pull request merged**